backend/src/test/java/com/bakdata/conquery/integration/IntegrationTests.java

```diff
@@ -77,64 +96,65 @@ public IntegrationTests(String defaultTestRoot, String defaultTestRootPackage) {
 			return Collections.singletonList(collectTests(reduced));
 		}
 		return reduced.getChildren().values().stream()
-			.map(this::collectTests)
-			.collect(Collectors.toList());
+					  .map(this::collectTests)
+					  .collect(Collectors.toList());
 	}
 
 	@SneakyThrows
 	public Stream<DynamicNode> programmaticTests() {
 		List<Class<?>> programmatic = CPSTypeIdResolver
-			.SCAN_RESULT
-			.getClassesImplementing(ProgrammaticIntegrationTest.class.getName())
-			.filter(info -> info.getPackageName().startsWith(defaultTestRootPackage))
-			.loadClasses();
+											  .SCAN_RESULT
+											  .getClassesImplementing(ProgrammaticIntegrationTest.class.getName())
+											  .filter(info -> info.getPackageName().startsWith(defaultTestRootPackage))
+											  .loadClasses();
```

Comment:
Ich glaube das Einrücken auf Überhang funktioniert nur gut wenn man nur Spaces benutzt. Sonst ist es hier immer komisch lesbar weil die Einrückungen irgendwo sind und dann auch noch aus Tabs und Spaces bestehen.

Kannst du das bitte deaktivieren?

Reply:
Diese letzten paar Leerzeichen habe ich schon seit Jahren versucht zu deaktivieren und nicht hinbekommen, hast du dafür das entsprechende Flag gefunden? Und Intellij scheint das hier irgendwie gar nicht zu schaffen:

```
		List<Class<?>> programmatic = CPSTypeIdResolver.SCAN_RESULT
											  .getClassesImplementing(ProgrammaticIntegrationTest.class.getName())
											  .filter(info -> info.getPackageName().startsWith(defaultTestRootPackage))
											  .loadClasses();
```

ist die "optimale Formattierung" oO

Reply:
Du hast sicher dieses Flag aktiviert und dass lässt es nicht zu, dass der Tabstop irgendwo in der "Nähe" des Zeilenendes verbleibt.
![image](https://user-images.githubusercontent.com/12283268/128172288-508a12a9-5b84-4d20-a3e9-bf44cba3cd05.png)

Reply:
danke, das wars